### DISCUSSION TITLE
v1.2.9

### DISCUSSION THREAD

**Discussion by zippity21** [zippity21/Stardew_Valley_Regression_Mod#87] [Announcements]
After letting this collect dust for nearly a year, I'm doing some quick house-cleaning (merge the long-pending PRs). Hopefully I can find some time when v1.6 comes out.

## What's Changed
* Typo Fixes by @klorpa in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/73
* Spacing and Spelling Fixes by @klorpa in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/76
* More Toilets by @DaveDipper in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/72
* Oniryuuko main by @zippity21 in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/85

## New Contributors
* @klorpa made their first contribution in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/73
* @DaveDipper made their first contribution in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/72

**Full Changelog**: https://github.com/zippity21/Stardew_Valley_Regression_Mod/compare/v1.2.8...v1.2.9

<hr /><em>This discussion was created from the release <a href='https://github.com/zippity21/Stardew_Valley_Regression_Mod/releases/tag/v1.2.9'>v1.2.9</a>.</em>

**Comment by fingerbangg** [comment]
Hey! I tried to get this mod working with the versions of everything that were listed but the SMAPI startup is telling me that the PyTK isn't compatible anyway, which prevents this mod from loading since it relies on that. I double checked the versions to make sure they were the ones listed (and they were - none of my current version mods loaded either so I also know SMAPI wasn't the issue) but it still just said it was incompatible. Do you think it would be possible to just update this mod so that it's compatible with the latest versions of SMAPI and PyTK?

**Reply by Wimsik** [reply]
Just posted a fix into another thread over in Issues, lemme copy and paste here for ya. As a note to the dev(s), might be worth updating the installation instructions to specify downloading the original version, at least until a Lite-compatibility update releases.

Ran into this while helping a friend install the other day as well. In our case, the out of date error was due to downloading PyTK Lite instead of PyTK Original. Give these steps a shot and see if they work:

1. Remove/Delete your current PyTK folder from your \Stardew Valley\Mods folder
2. Go here: https://www.nexusmods.com/stardewvalley/mods/1726?tab=files
3. Download the first thing under Optional Files, PyTK - Platonymous Toolkit/"Original version".
4. Unzip the PyTK folder into your \Stardew Valley\Mods folder
5. Try and launch through SMAPI again.

Hope this helps!

**Comment by JadeTheProtogen** [comment]
I've followed the instructions (Including the thing you suggested for them ^) but it won't activate.

**Reply by zippity21** [reply]
Version 1.6 of Stardew Valley has recently been released, and it contains a number of changes regarding mods. I have a compatibility update in the works, but PyTk doesn't yet have a corresponding update.

Unfortunately, this means I'm stuck until the PyTk author updates, or I find a way to drop it as a dependency (non-trivial).

**Reply by OriginCosdenex15** [reply]
I did get the Lite version be activated when I use it with 1.6, but I haven't tested it with any PyTK mods.

**Reply by zippity21** [reply]
Unfortunately, the Lite version doesn't provide the ISavable interface, which this mod uses. I'm trying to remove the dependency, but it is non-trivial and I have very little time to work on it. Hopefully the full version updates.

**Reply by OriginCosdenex15** [reply]
What is planned for the next update, is is just a port or will there be new content?
Also how far is the porting to remove PyTK as a dependency progress?
